tests/test_py_residue.py

```
@@ -0,0 +1,427 @@
+    assert empty.one_letter_code == ""
+
+
+def test_py_residue_special_amino_acids():
+    """Test special amino acids like selenocysteine."""
+    # Note: Availability depends on ResidueDB configuration
+    # This test might need adjustment based on your OpenMS version
+    try:
+        # Try to get selenocysteine (U)
+        sec = Py_Residue.from_string("U")
+        assert sec.one_letter_code == "U"
+    except:
```

Ruff (0.14.8)

Do not use bare `except`

(E722)